### ISSUE TITLE
FYI - Node.js calendar lost - possibly gmail cleanup

### ISSUE THREAD

**Issue by mhdawson** [nodejs/admin#923]
The Node.js calendar and all entries disapeared over the weekend.

We are not sure why, possibly due to gmail cleanup.

Turns out we did not have the credentials for the user who had created the calendar anywhere I could find (build secrets, 1 password).

The foundation has now created a shared calender under their account and I've repopulated the calender entries. If I've missed any please let us know and I'll add them.

There may be a small tweak to https://github.com/nodejs/admin#calendar-maintainers in terms of how we manage access for those who can add/edit/delete entries but @bensternthal is still figuring that out.

I did take the opportunity to given @bensternthal  a different list than was on the admin page to have access as it was quite out of date and had people I know no longer had access and was missing some who did. If I missed including you in the new list just let me know and we'll get that fixed up.

**Comment by mhdawson** [comment]
@joesepi @ovflowd FYI in case you have any other info/comments to add to the issue.

**Comment by mhdawson** [comment]
@ovflowd can you confirm once the nodejs.org/calendar redirect is updated?

**Comment by ovflowd** [comment]
This sounds like a good summary of the events. Maybe we should add a timeline of what we discovered and how things happened?

Maybe also reference to the Slack messages?

**Comment by ovflowd** [comment]
> @ovflowd can you confirm once the nodejs.org/calendar redirect is updated?

I don't have access to that. I believe the Build WG has write access to Cloudflare to update that. cc @nodejs/build

**Comment by mhdawson** [comment]
@ovflowd are you shure it is configured in Cloudflare and not just a website redirect?  I looked in cloudflare and so far have not found it any any of the likely places.

**Comment by ovflowd** [comment]
> @ovflowd are you shure it is configured in Cloudflare and not just a website redirect?  I looked in cloudflare and so far have not found it any any of the likely places.

I don't recall, but it could be inside redirects.json -- Im not close to a PC right now, so if you could check on the repo in apps/website/redirects.json thatd be great!

**Comment by richardlau** [comment]
https://github.com/nodejs/nodejs.org/blob/10aa14912ee9a3e0cdab12896c906368b7b4f46c/apps/site/redirects.json#L15-L18

**Comment by richardlau** [comment]
> Looks to be defined here: https://github.com/nodejs/nodejs.org/blob/10aa14912ee9a3e0cdab12896c906368b7b4f46c/apps/site/redirects.json#L15-L22

PR: https://github.com/nodejs/nodejs.org/pull/7098

You can check the redirect, e.g. https://nodejs-org-git-updatecalendar-openjs.vercel.app/calendar (based on the preview link for https://github.com/nodejs/nodejs.org/pull/7098 being https://nodejs-org-git-updatecalendar-openjs.vercel.app/)

**Comment by mhdawson** [comment]
Looks like the PR to fix nodejs.org/calendar has landed.

**Comment by bensternthal** [comment]
@mhdawson I have given you permission to manage sharing, so I think from my perspective this is all set.

**Comment by mhdawson** [comment]
@bensternthal am I going to be able to pass that permission on to other TSC members?

**Comment by bensternthal** [comment]
@mhdawson I believe so yes! Do test to be sure.

**Comment by ovflowd** [comment]
BTW, are we going to fix the Node.js Calendar here on Slack? 

![image](https://github.com/user-attachments/assets/1f31c7f0-2216-44a8-8080-4a47647b7cf2)

**Comment by mhdawson** [comment]
@ovflowd any idea who set that up? The link is ok but I guess there is some integration that is now looking at the wrong calendar?

**Comment by richardlau** [comment]
I don't know who set the channel/app up, but events to that channel were posted by 
![image](https://github.com/user-attachments/assets/93a60ee0-a1e0-4376-8125-fd1c10b05520)

"About this app" gives
![image](https://github.com/user-attachments/assets/9e7545c7-bdd3-46f2-9ab5-38f5ca7084e3)

which suggests that this may have stopped working prior to the recent issues with the calendar.

**Comment by ovflowd** [comment]
Also no idea, @bensternthal can you help us setting up the calendar there?

**Comment by bensternthal** [comment]
I agree with @richardlau it appears the app was discontinued on September 10th. Here’s what I see when I attempt to access the configuration:

<img width="839" alt="Screenshot 2024-10-23 at 10 36 34 AM" src="https://github.com/user-attachments/assets/d32005a0-845e-498d-b16f-053e05c8a238">

There is a suggested Google Calendar app available, but it seems to be linked to an individual’s Gmail account and only uses their primary calendar. It is not a substitute for the app we have been using. 

Does anyone have a solution that they use elsewhere?

**Comment by bensternthal** [comment]
The same integration and just like Node, updates stopped around September 10th.

**Comment by ovflowd** [comment]
> The same integration and just like Node, updates stopped around September 10th.

I believe there must be some other sort of integration to show calendars. It even says there, for alternative, use "Google Calendar app"

**Comment by bensternthal** [comment]
@ovflowd see my previous note:

_"There is a suggested Google Calendar app available, but it seems to be linked to an individual’s Gmail account and only uses their primary calendar. It is not a substitute for the app we have been using."_

I wanted to see if folks had a known solution for this before any of us burn time looking for an alternative.

**Comment by ovflowd** [comment]
> @ovflowd see my previous note:
> 
> _"There is a suggested Google Calendar app available, but it seems to be linked to an individual’s Gmail account and only uses their primary calendar. It is not a substitute for the app we have been using."_
> 
> I wanted to see if folks had a known solution for this before any of us burn time looking for an alternative.

Oh I missed that 🙈

**Comment by ovflowd** [comment]
@bensternthal wdyt -- https://www.vendr.com/blog/how-do-you-add-a-calendar-to-slack

**Comment by bensternthal** [comment]
@ovflowd this looks like an article written solely for SEO. It does mention "Eventbot Calendar" which has a paid option of 26 a month with a 50% discount for non-profits.  That's not a lot, however they have a concept of "Eventbot users" and when you click the link https://support.geteventbot.com/hc/en-us/articles/360007162634 it 404s.

So I am not sure if this company is legit, many links 404. 

But I am questioning if this is needed. There is a public calendar that is easy to subscribe to and receive alerts, and this certainly is easiest method and does not rely on a third party.
